### PR TITLE
[10.0][MIG] base_import_odoo

### PR DESCRIPTION
**Pull request by hbrunn**
relevant commit is https://github.com/OCA/server-tools/pull/1319/commits/975a4e40d60d24142698c69b798283e6c9ab90f7 and I took #1294 with me too

### CODE REVIEW

File: base_import_odoo/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from . import ir_model_data
```
Comment (hieulucky111) [inline base_import_odoo/models/__init__.py]: import order

File: base_import_odoo/models/import_odoo_database.py [hunk 2]
```diff
@@ -0,0 +1,685 @@
...
+try:
+    import odoorpc
+except ImportError:  # pragma: no cover
+    logging.info('Unable to import odoorpc, used in base_import_odoo')
+    odoorpc = False
+
+import psycopg2
+import traceback
+from urlparse import urlparse
+from openerp import _, api, exceptions, fields, models, tools
```
Comment (hieulucky111) [inline base_import_odoo/models/import_odoo_database.py]: i think we should use odoo instead openerp.

**Review by sebalix — CHANGES_REQUESTED** [review]
s/openerp/odoo/ in the import statements is welcome

**Comment by hbrunn** [comment]
@hieulucky111 @sebalix thanks for the input, I fixed those back then, but didn't ping you. Could you re-review?

File: base_import_odoo/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Import from Odoo",
+    "version": "10.0.1.0.1",
```
Comment (sebalix) [inline base_import_odoo/__manifest__.py]: No need to reset to `10.0.1.0.0`?

**Review by sebalix — APPROVED** [review]
Minor comment, LG!

File: base_import_odoo/__manifest__.py (hunk 3 above)
Comment (hbrunn) [reply base_import_odoo/__manifest__.py]: thanks, did that

File: base_import_odoo/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Import from Odoo",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Tools",
+    "summary": "Import records from another Odoo instance",
```
Comment (tbaden) [inline base_import_odoo/__manifest__.py:10]: please add website tag with value https://github.com/OCA/server-tools

**Review by tbaden — CHANGES_REQUESTED** [review]
code review

File: base_import_odoo/__manifest__.py (hunk 4 above)
Comment (hbrunn) [reply base_import_odoo/__manifest__.py:10]: thanks, done

**Review by tbaden — APPROVED** [review]

**Comment by rousseldenis** [comment]
@OCA/core-maintainers

**Pull request closed without merging**